SendWithUs.Client.Tests/EndToEnd/BatchAsyncTests.cs

```diff
@@ -30,22 +30,32 @@ namespace SendWithUs.Client.Tests.EndToEnd
     public class BatchAsyncTests
     {
         [TestMethod]
-        public void BatchAsync_OneSendRequest_Succeeds()
+        public void BatchAsync_TwoRequests_Succeeds()
         {
+            // Arange
             var subject = "BatchAsync " + TestHelper.GetUniqueId();
             var testData = new TestData("EndToEnd/Data/SendRequest.xml");
-            var request = new SendRequest(testData.TemplateId, testData.RecipientAddress, testData.Data.Upsert("subject", subject));
+            var sendRequest = new SendRequest(testData.TemplateId, testData.RecipientAddress, testData.Data.Upsert("subject", subject));

```

Comment:
All EndToEnd tests fail when running the public distribution. This is intentional. Otherwise, I'd have to expose a valid API key in SendRequest.xml.